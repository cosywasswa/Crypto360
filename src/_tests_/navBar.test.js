import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';

describe('test the navBar Component', () => {
  it('should render navBar correctly', () => {
    const tree = renderer.create(
      <BrowserRouter>
        <navBar />
      </BrowserRouter>,
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
