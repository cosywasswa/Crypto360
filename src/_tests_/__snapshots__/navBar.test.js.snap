// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test the navBar Component should render navBar correctly 1`] = `<navBar />`;
